ossdbtoolsservice/query/batch.py

```diff
@@ -169,8 +183,8 @@ def get_cursor(self, connection: ServerConnection):
         # and we explicitly close it we are good
         return connection.cursor(name=cursor_name, withhold=True)
 
-    def after_execute(self, cursor) -> None:
-        super().create_result_set(cursor)
+    def after_execute(self, cursor, cancellationToken: CancellationToken) -> None:
```

Comment:
Use python naming for parameters.
it should be cancellation_token.
We use underscore instead of caps.

Fix it in the complete code.

Reply:
done